scripts/whitelist-fiscal-code-script/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "whitelist-fiscal-code-script",
+  "version": "0.0.1-rc.1",
+  "scripts": {
+    "build": "tsc",
+    "start": "ts-node src/main.ts",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "test": "vitest run",
+    "test:coverage": "vitest run --coverage",
+    "format": "prettier --write ."
+  },
+  "engines": {
+    "node": ">=20.17.0",
+    "npm": ">=10.8.2"
+  },
+  "description": "This script is used to whitelist fiscal codes in the database.",
```

This script adds fiscal codes to the whitelist in the database.
```suggestion
  "description": "This script adds fiscal codes to the whitelist in the database.",
```